lib/takeaway.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'dish.rb'
+require_relative 'text.rb'
+require 'twilio-ruby'
+
+class Takeaway 
+  attr_reader :menu, :final_order, :texter
+  def initialize
+    @texter = Text.new
+    @menu = {
+      "dish 1" => 5,
+      "dish 2" => 7,
+      "dish 3" => 9}
+    @final_order = []
+  end
+
+  def menu
```

Comment:
A good, concise and readable method but could maybe have a more descriptive name seeing as it displays data; you also have an @menu attr_reader which might be confusing when calling Takeaway.menu